cmd/meroxa/turbine/javascript/upgrade.go

```diff
@@ -0,0 +1,45 @@
+package turbinejs
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/meroxa/cli/log"
+)
+
+// Upgrade fetches the latest Meroxa dependencies.
+func (t *turbineJsCLI) Upgrade(appPath string, vendor bool) error {
+	cmd := exec.Command("grep", "turbine-js", "package.json")
+	cmd.Dir = appPath
+	err := cmd.Wait()
+	if err != nil {
+		t.logger.StartSpinner("\t", "Adding @meroxa/turbine-js requirement...")
+		cmd = exec.Command("npm", "install", "@meroxa/turbine-js", "--save")
```

Comment:
these references can now be updated to be `@meroxa/turbine-js-framework`

Reply:
do i have to deal with the case where turbine-js is already in package.json?

Reply:
Yeah to keep things tidy for the small period of turbine-js dependency installs… but this should just be as easy as adding an “npm uninstall @meroxa/turbine-js” call before the other npm commands and eventually we can remove.